src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -47,8 +47,9 @@ final class PublishMercureUpdatesListener
     private $createdEntities;
     private $updatedEntities;
     private $deletedEntities;
+    private $formats;
 
-    public function __construct(ResourceClassResolverInterface $resourceClassResolver, IriConverterInterface $iriConverter, ResourceMetadataFactoryInterface $resourceMetadataFactory, SerializerInterface $serializer, MessageBusInterface $messageBus = null, callable $publisher = null, ExpressionLanguage $expressionLanguage = null)
+    public function __construct(ResourceClassResolverInterface $resourceClassResolver, IriConverterInterface $iriConverter, ResourceMetadataFactoryInterface $resourceMetadataFactory, SerializerInterface $serializer, MessageBusInterface $messageBus = null, callable $publisher = null, array $formats, ExpressionLanguage $expressionLanguage = null)
```

Comment:
Is there a way to extract the formats from `OperationAwareFormatsProviderInterface` instead of using the config as formats can be defined at operation/resource level ?

Reply:
thanks for the hint :) 👍